src/code.sh

```
@@ -8,32 +8,33 @@ dx-download-all-inputs
 lib_dir=$(find /home/dnanexus/genome_lib -type d -name "*" -mindepth 1 -maxdepth 1 | rev | cut -d'/' -f-1 | rev)
 
-if ! tar -xzf /home/dnanexus/in/arriba_tar/*.tar.gz -C /home/dnanexus/ --one-top-level=arriba; then
-    echo "Error: Failed to extract arriba library"
-    exit 1
-fi
-
-cd arriba && make
-cd ..
-export PATH=$PATH:/home/dnanexus/arriba
+# Load arriba docker
+docker load -i $arriba_tar_path
```

Shellcheck (0.10.0)

[warning] arriba_tar_path is referenced but not assigned.

(SC2154)